common/culture/traditions/00_regional_traditions.txt

```diff
@@ -76,7 +90,7 @@ tradition_lords_of_the_elephant = {
 			# Warcraft
 			AND = {
 				any_culture_county = {
-					title_province = {
+					any_county_province = {
 						geographical_region = world_innovation_elephants
 					}
 				}
```

Comment:
It was a bug? Any is basically a cycle. So it doesn't look good for performance.

Reply:
Good catch; didn't know that. I'll try and adjust it.

Reply:
Adjusted with e3f38ac and 6cb3520